test/unit/annotation_spec.js

```diff
@@ -2280,6 +2387,65 @@ describe("annotation", function () {
         }, done.fail);
     });
 
+    it("should render radio for printing using normal appearance", function (done) {
```

Comment:
`should render radio buttons for printing using normal appearance` ("buttons" should be added).